plugin/src/main/java/io/jenkins/plugins/casc/ConfigurationAsCode.java

```diff
@@ -556,7 +557,23 @@ public void doViewExport(StaplerRequest2 req, StaplerResponse2 res) throws Excep
         ByteArrayOutputStream out = new ByteArrayOutputStream();
         export(out);
 
-        req.setAttribute("export", out.toString(StandardCharsets.UTF_8.name()));
+        req.setAttribute("viewExport", new ManagementLink() {
```

Comment:
I've been doing a dirty in Credentials plugin and just using a class that has a displayName...